libs/blocks/caas-config/caas-config.js

```diff
@@ -68,6 +77,10 @@ const defaultOptions = {
     '32Margin': '32 Margin Container',
     carousel: 'Carousel',
   },
+  ctaActions: {
+    '_blank': "blank",
+    '_self': "self",
```

Comment:
use single quotes

Reply:
Done. Surprised since I ran the linter before committing and it wasn't caught. Maybe linter isn't catching these?